client/src/components/Pages/Structure/Input/TxSegmentElementInput/TxSegmentElementInput.tsx

```diff
@@ -89,30 +89,20 @@ const TxSegmentCompInput: React.FC<TxSegmentElementInputProps> = ({
 
   const [pendingResponse, setPendingResponse] = useState(false);
 
-  /*
-  Depending on this element's location in the structure array, the user
-  needs to provide some kind of coordinate input for either one or both ends
-  of the element. This can change as the user drags the element around the structure
-  array, or adds other elements to the array.
-  */
   const hasRequiredEnds =
-    index !== 0 && index < fusion.length
-      ? (txStartingGenomic && txEndingGenomic) || (startingExon && endingExon)
-      : index === 0
-      ? txEndingGenomic || endingExon
-      : txStartingGenomic || startingExon;
+    txEndingGenomic || endingExon || txStartingGenomic || startingExon;
```

Comment:
please double check me on this change! From my understanding, based on the sequence location changes, a Tx segment element only requires either a start/end, regardless of position (it looks like previously, it had to have both unless it was the first item?)

Reply:
Yes one of start/end is required to create a TranscriptSegmentElement